src/main/webapp/app/exercises/programming/manage/programming-exercise-detail.component.html

```diff
@@ -272,7 +272,11 @@ <h2><span jhiTranslate="artemisApp.exercise.details">Exercise Details</span></h2
                     <jhi-exercise-details [exercise]="programmingExercise"></jhi-exercise-details>
                     <dt><span jhiTranslate="artemisApp.exercise.feedbackSuggestionsEnabled">Enable feedback suggestions from Athena</span></dt>
                     <dd>
-                        <span>{{ (programmingExercise.feedbackSuggestionsEnabled ? 'global.generic.yes' : 'global.generic.no') | artemisTranslate }}</span>
+                        <span>{{
```

Comment:
Would using if/else here make sense to be consistent with course-detail.component?